apps/src/templates/sectionProgress/SectionProgress.jsx

```diff
@@ -146,6 +149,10 @@ class SectionProgress extends Component {
                 studentLevelProgress={studentLevelProgress}
                 lessonOfInterest={this.state.lessonOfInterest}
               />
+              <LoadMoreProgressButton />
```

Comment:
Do we need <LoadMoreProgressButton/> in both places?  Can we move it to the bottom of the page since it will show in both views?

Reply:
Added it to both since we want it to show above the legend.